yogstation/code/modules/scripting/Errors.dm

```diff
@@ -111,13 +131,19 @@
 
 	UnknownInstruction
 		name="UnknownInstructionError"
-		message="Unknown instruction type. This may be due to incompatible compiler and interpreter versions or a lack of implementation."
+		New(node/op)
+			message="Unknown instruction type '[op.type]'. This may be due to incompatible compiler and interpreter versions or a lack of implementation."
```

Comment:
It might be a good idea to set up something that gives a warning whenever a ``process_signal`` function is not defined in the script, to help stop people reeing at NTSL being "broken."